chapters/methodology.tex

```diff
@@ -62,6 +62,22 @@ \subsection{Architecture}
 allows easy databinding to the UI (view), whilst allowing easy access from the
 back end (model). \cite{mvvm}
 
+\subsection{Data Collection} \label{sec:data_collection}
+
+As shown in \ref{fig:arch1}, the image analyser produces \texttt{DetectionResult}s.
+In order to collect multiple \texttt{DetectionResult}s, a
+class called \texttt{Measurement} gathers multiple detection results as

```

Comment:
Did you intend to write "detection results" instead of "`\texttt{DetectionResult}s`"?